src/components/search/Search.jsx

```diff
@@ -44,6 +45,7 @@ const Search = (props) => {
       <div id="search">
         <Header triggerEditorMenu={props.triggerHandleOffsetMenu} />
         <ContextAlert />
+        <PreviewModal />
```

Comment:
I realized this was in the wrong place -- it needed to be higher in the component hierarchy.